### ISSUE TITLE
Missing newComputePipelineState variant

### ISSUE THREAD

**Issue by kvark** [madsmtm/objc2#683]
This one appears to be missing - https://developer.apple.com/documentation/metal/mtldevice/1433353-newcomputepipelinestatewithdescr
The only variant with a descriptor that I see is `newComputePipelineStateWithDescriptor_options_completionHandler`, which is a different signature.

**Comment by madsmtm** [comment]
You're right, it's intentionally skipped [here](https://github.com/madsmtm/objc2/blob/cd299d223fec1ee26e94ba9bd91fb08c357f63ad/framework-crates/objc2-metal/translation-config.toml#L14-L24).

I think I have a better idea of how to handle such cases such as `MTLAutoreleasedComputePipelineReflection` nowadays, lemme try to fix it.

**Comment by madsmtm** [comment]
I've fixed this now in 8c383ce079ef15f560493d55a0ebe41ea0fc4004, thanks for the prompt to do so.
